pkg/fixedpoint/dec.go

```diff
@@ -1061,6 +1061,9 @@ func (v *Value) UnmarshalYAML(unmarshal func(a interface{}) error) (err error) {
 
 // FIXME: should we limit to 8 prec?
 func (v Value) MarshalJSON() ([]byte, error) {
+	if v.IsInf() {
+		return []byte("\"" + v.String() + "\""), nil
```

Comment:
you can use strconv.Quote here

Reply:
We don't need to escape the quotation, seems like overkilled